examples/delete-file.js

```diff
@@ -0,0 +1,51 @@
+const {
+  Client,
+  FileCreateTransaction,
+  FileDeleteTransaction,
+  Ed25519PublicKey,
+  Hbar
+} = require("@hashgraph/sdk");
+
+async function main() {
+
+  const operatorAccount = process.env.OPERATOR_ID;
+  const operatorPrivateKey = process.env.OPERATOR_KEY;
+  const operatorPublicKey = Ed25519PublicKey.fromString(process.env.OPERATOR_PUB_KEY);
+
+  if (operatorPrivateKey == null || operatorAccount == null) {
+    throw new Error(
+      "environment variables OPERATOR_KEY and OPERATOR_ID must be present"
+    );
+  }
+
+  const client = new Client({
+    network: { "0.testnet.hedera.com:50211": "0.0.3" },
+    operator: {
+      account: operatorAccount,
+      privateKey: operatorPrivateKey
+    }
+  });
+
+  // First, we'll create a file with our operator as an admin
+  const transactionId = await new FileCreateTransaction()
+    .setContents("creating a file to test deletion")
+    .setMaxTransactionFee(Hbar.of(100))
```

Comment:
It should only cost ~2 hbar to create a file. For exposition we should try and keep the max declared fee close to what it costs.